integration/loki_micro_services_delete_test.go

```diff
@@ -115,4 +117,49 @@ func TestMicroServicesDeleteRequest(t *testing.T) {
 		require.Equal(t, `{job="fake"} |= "lineB"`, deleteRequests[0].Query)
 		require.Equal(t, "received", deleteRequests[0].Status)
 	})
+
+	// Wait until delete request is finished
+	t.Run("wait-until-delete-request-processed", func(t *testing.T) {
+		require.Eventually(t, func() bool {
+			deleteRequests, err := cliCompactor.GetDeleteRequests()
+			require.NoError(t, err)
+			require.NotEmpty(t, deleteRequests)
```

Comment:
Nit: `require.NotEmpty` is redundant if we're `require.Len(t,...,1)` on the next line

Reply:
Indeed. That check has been removed.